__tests__/app.test.js

```diff
@@ -112,3 +112,53 @@ describe("GET /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("200: returns array of comments associated with valid specified review_id", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        expect(comments.length).toBe(3);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            review_id: 2,
+          });
+        });
+      });
+  });
+  test("200: returns empty array when given a review_id that exists but has no associated comments", () => {
+    return request(app)
+      .get("/api/reviews/1/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        expect(response.body).toHaveProperty("comments");
+        expect(comments.length).toBe(0);
+      });
+  });
+  test("404: returns an error when given review_id that does not exist", () => {
+    return request(app)
+      .get("/api/reviews/150/comments")
+      .expect(404)
+      .then((response) => {
+        const { msg } = response.body;
+        expect(msg).toBe("There is no user with that id");
```

Comment:
This error message is not clear - why "no user with that id"? 
The id is not referring to a user at all, this should be changed to a more appropriate error message